.chloggen/signaltometrics-use-ottl-valueexpression.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: signaltometricsconnector
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: `get` OTTL function is removed and expressions are now parsed using `ParseValueExpression`
```

Oh you need to escape the backtick character 
```suggestion
note: "`get` OTTL function is removed and expressions are now parsed using `ParseValueExpression`"
```

---

Ah, Thanks!